src/Umbraco.Cms.Integrations.DAM.Aprimo/AprimoComposer.cs

```diff
@@ -39,7 +39,8 @@ public void Compose(IUmbracoBuilder builder)
                     client.DefaultRequestHeaders.Add("API-VERSION", "1");
                     client.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue("Umbraco.Cms.Integrations.DAM.Aprimo", "1.0.0"));
                     client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
-                    client.DefaultRequestHeaders.Add("select-record", "title,tag,thumbnail");
+                    client.DefaultRequestHeaders.Add("select-record", "title,thumbnail,fields,masterfilelatestversion");
```

Comment:
My guess would be that these two aren't needed for all request, just the one where you get an asset by Id?

If so, they probably shouldn't be default headers sent on all requests, rather added to the `Headers` collection for just the request that needs it.